Britannia/localization/english/BTWK_nicknames_l_english.yml

```diff
@@ -25,7 +25,8 @@
  nick_old_hawk:0 "the Old Hawk"
  nick_strongboar:0 "Strongboar"
  nick_hadrianus:0 "Hadrianus"
- 
+ nick_hadrianus_desc:1 "Meaning #italic of Hadrian#, [CHARACTER.GetShortUINameNoTooltipNoFormat|U] [CharAreIs(CHARACTER)] hailed as a worthy successor to the great Roman Emperor Hadrian, thanks to [CharYourHerHis(CHARACTER)] efforts in restoring the wall of the same name. " 
```

Comment:
Meaning #italic of Hadrian# - Might want to remove this part as Hadrianus is the actually just the Latin for Hadrian. If you wanted to actually make the nickname mean "of Hadrian" you probably want something like "Hadriani" (sorry I'm not that familiar with Latin) but I think just straight up nicknaming the character as Hadrian(us) makes more sense.

Reply:
Don't worry, I'm not familiar with Latin either. For that, all I did was type "of Hadrian" into an English to Latin translator and it came back as "Hadrianus". I suppose the point of the nickname would be that the ruler has emulated Hadrian(us), so (despite my original method coming about it being wrong) I think the nickname is fine, yes?

Reply:
I think a nickname meaning "of Hadrian" doesn't make sense in this context - consider how it sounds in English, e.g. High King Arthur 'of Hadrian', which TBH just sounds like he comes from a place called Hadrian.